<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_sliders</class>
 <widget class="QDialog" name="Dialog_sliders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>148</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>399</width>
    <height>148</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>399</width>
    <height>148</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="1" column="0" colspan="2">
    <widget class="QSlider" name="slider_value1">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="spin_value2">
     <property name="suffix">
      <string/>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QSlider" name="slider_value2">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="spin_value1">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="btn_cancel">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_value1">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_value2">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="btn_ok">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spin_value1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_value1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_value2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_value2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_value1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_value1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_value2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_value2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
